### PR TITLE
Add Additional Property-Based Tests for LinkedList

### PR DESCRIPTION
**Pull request by alipourhabibi**
Overview:
This pull request adds new Property-Based Tests using testing/quick library.

Changes:
- Add Property-Based Tests for LinkedList

How Has This Been Tested?
- Ran the new tests locally and ensured they pass
- Verified that the existing tests still pass

### CODE REVIEW

**Comment by Iajrdev** [comment]
> Overview: This pull request adds new Property-Based Tests using testing/quick library.
> 
> Changes:
> 
> * Add Property-Based Tests for LinkedList
> 
> How Has This Been Tested?
> 
> * Ran the new tests locally and ensured they pass
> * Verified that the existing tests still pass

@alipourhabibi 
Please rename the test file to Linkedlist_test.go and update the test commands in the README to reflect these latest changes.

**Comment by alipourhabibi** [comment]
I put all tests in one file and fixed the ci

File: linkedList_test.go [hunk 1]
```diff
@@ -2,8 +2,125 @@ package main
 
 import (
 	"testing"
+	"testing/quick"
+
+	"github.com/leanovate/gopter"
+	"github.com/leanovate/gopter/gen"
+	"github.com/leanovate/gopter/prop"
```
Comment (Iajrdev) [inline linkedList_test.go]: Remove the following imports:
```
"github.com/leanovate/gopter"
"github.com/leanovate/gopter/gen"
"github.com/leanovate/gopter/prop"
```

File: linkedList_test.go [hunk 2]
```diff
@@ -2,8 +2,125 @@ package main
...
+					return false
+				}
+			}
+			return true
+		},
+		gen.SliceOf(gen.Int()),
+	))
+
+	properties.TestingRun(t)
+}
```
Comment (Iajrdev) [inline linkedList_test.go]: The function TestLinkedListProperties based on the `gopter` library it should be removed. We already use the quick package for property-based testing in the same file. To maintain consistency and avoid redundancy please remove this part

**Review by Iajrdev — CHANGES_REQUESTED** [review]
Please change highlighted sections in test file

**Comment by alipourhabibi** [comment]
I rebased the commits to one commit

**Review by Iajrdev — APPROVED** [review]
I've reviewed the code and everything looks good to me.

**Pull request merged**